### PR TITLE
Instructions for other distros

### PR DESCRIPTION
**Pull request by lilkidsuave**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,4 +17,54 @@ Aside from these lines, all the other code in this repository has been written b
...
+Copy the following to eithier /home/(user)/.local/bin or /usr/bin
+
+blend-files
+blend-system
+blend.hook
+blend.install
+host-blend
+init-blend 
+blend-settings
+
```
Comment (antonio24073) [inline README.md:38]: blend

File: README.md [hunk 2]
```diff
@@ -17,4 +17,54 @@ Aside from these lines, all the other code in this repository has been written b
 
 ## Usage
 
-It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
\ No newline at end of file
+It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
+
+#Installation
+install podman from your distro's repo.
+i.e sudo dnf install podman
```
Comment (antonio24073) [inline README.md:24]: pip install pexpect
cd blend-settings
npm i

File: README.md [hunk 3]
```diff
@@ -17,4 +17,54 @@ Aside from these lines, all the other code in this repository has been written b
...
+init-blend 
+blend-settings
+
+sudo cp blend-files blend-system blend.hook blend.install host-blend init-blend blend-settings
+
+Copy the service file to /etc/systemd/system
+sudo cp blend-files.service /etc/systemd/system
+
+Take ownership of /usr/libexec/initcripts
+sudo chown (user) -R /usr/libexec/initscripts
```
Comment (antonio24073) [inline README.md:51]: this is not needed in manjaro gnome 43

File: README.md [hunk 4]
```diff
@@ -17,4 +17,54 @@ Aside from these lines, all the other code in this repository has been written b
...
+                'version': 'version' }
+                'command', choices=command_map.keys(), help=argparse.SUPPRESS)
+                'pkg', action='store', type=str, nargs='*', help=argparse.SUPPRESS)
+                '-cn', '--container-name', action='store', nargs=1, metavar='CONTAINER NAME', help=argparse.SUPPRESS)
+                '-y', '--noconfirm', action='store_true', help=argparse.SUPPRESS)
+                '-d', '--distro', action='store', nargs=1, metavar='DISTRO', help=argparse.SUPPRESS)
+                '-v', '--version', action='version', version=f'%(prog)s {__version}', help=argparse.SUPPRESS)
+           
+             
+
```
Comment (antonio24073) [inline README.md:108]: to execute GUI (in the project root folder):
```
electron ./blend-settings
```

it's working fine in manjaro but I put in the container name `ubuntu` and it started `arch linux`

File: README.md (hunk 3 above)
Comment (lilkidsuave) [reply README.md:51]: This was needed in Fedora 38
Your mileage may vary

File: README.md (hunk 1 above)
Comment (lilkidsuave) [reply README.md:38]: Ah missed it thanks

File: README.md (hunk 4 above)
Comment (lilkidsuave) [reply README.md:109]: Don't need to do that

Comment (lilkidsuave) [reply README.md:109]: Move the gui files to where they go so you can run in terminal no matter where you are.

Comment (lilkidsuave) [reply README.md:121]: That's the whole point vs using distrobox 

The seamless ness

File: README.md (hunk 2 above)
Comment (lilkidsuave) [reply README.md:24]: ah i see

this is useful for gui install :)
but i started with cli, and the cli instructions dont get npm until installing node.js

File: README.md (hunk 4 above)
Comment (antonio24073) [reply README.md:122]: @lilkidsuave Thank you, I couldn't have done it without you. not so easily

Comment (lilkidsuave) [reply README.md:124]: Thank you for the feedback and helping me test : )

**Pull request closed without merging**